src/Label/Commands/Create.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace CultuurNet\UDB3\Label\Commands;
+
+use CultuurNet\UDB3\Label\ValueObjects\Privacy;
+use CultuurNet\UDB3\Label\ValueObjects\Visibility;
+use ValueObjects\Identity\UUID;
+use ValueObjects\String\String as StringLiteral;
+
+class Create extends AbstractCommand
```

Comment:
Had dit met Nick besproken en namespace leek ons voldoende, zie ook https://jira.uitdatabank.be/browse/III-945